frontend/src/components/common/CardField.tsx

```diff
@@ -0,0 +1,20 @@
+import { Text, VStack } from "@chakra-ui/react";
+import React from "react";
+
+interface CardFieldProps {
+  title: string;
+  value: string;
+}
+
+const CardField = ({ title, value }: CardFieldProps) => (
+  <VStack align="left">
+    <Text textTransform="uppercase" textStyle="mobileSmall" color="hubbard.100">
+      {title}
+    </Text>
+    <Text textStyle="mobileSmall" pb={["1rem", "0px"]}>
+      {value}
+    </Text>
+  </VStack>
+);
+
+export default CardField;
```

Comment:
could we remove this component and use https://github.com/uwblueprint/community-fridge-kw/blob/3fc83aa50c2bce8913ff25403513f1606c0e4f88/frontend/src/components/common/Card/index.tsx#L6